src/tests/degreeQuery.ts

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable no-console */
+import { expect } from 'chai';
+import {} from 'mocha';
+import supertest from 'supertest';
+
+describe('GraphQL', () => {
+    let request: any;
+
+    beforeEach(() => {
+        const url = 'http://localhost:3001';
+
+        request = supertest(url);
+    });
+
+    it('Returns degree with id = 1', done => {
+        request
+            .post('/graphql')
+            .send({
+                query: '{ degree(id: 1) { id nameJa nameEn abbreviation }}'
+            })
+            .expect(200)
+            .end((err: any, res: any) => {
+                // res will contain array with one user
+                if (err) { return done(err); }
+
+                console.log(res.body);
+                expect(res.body.data.degree).to.have.property('id').equal('1');
+                expect(res.body.data.degree).to.have.property('nameJa').equal('医師');
+                expect(res.body.data.degree).to.have.property('nameEn').equal('Medical Doctor');
+                expect(res.body.data.degree).to.have.property('abbreviation').equal('MD');
+
+                done();
+
+                return true;
+            });
+    });
+
+    it('Returns all degrees', done => {
+        request
+            .post('/graphql')
+            .send({
+                query: '{ degrees { id nameJa nameEn abbreviation }}'
+            })
+            .expect(200)
+            .end((err: any, res: any) => {
+                // res will contain array with one user
+                if (err) { return done(err); }
+
+                console.log(res.body);
+                expect(res.body.data.degrees).to.have.lengthOf(14);
```

Comment:
```suggestion
                expect(res.body.data.degrees).to.have.lengthOf(15);
```

Reply:
I ran the tests and this one failed because the length of the list of degrees is actually 15. I double checked it through a query in GraphQL.

Reply:
🤔 Be sure to start your DB from scratch (delete and remake the docker container). There are only 14 entries in degrees.csv. Perhaps you added one while doing local testing

Reply:
Just re-ran it on a fresh db, it's 14

Reply:
@ann-kilzer  Something strange is going on, because sometimes it'll pass and sometimes it won't. I deleted the docker container between each attempt, and out of 5 tries I got:

1. Failed 
2. Passed
3. Passed
4. Failed
5. Failed

Here's my most recent failing test:

<img width="881" alt="Screenshot 2023-01-29 at 10 33 37 PM" src="https://user-images.githubusercontent.com/31802656/215329686-09dea7ea-a615-4da9-868c-b7343a1ccb93.png">

I did a little bit of digging and Prisma Studio also only shows 13 degrees in the database. Something is happening when I create a new Docker container that is sometimes causing an incorrect number of degrees from being inserted it seems.
<img width="1065" alt="Screenshot 2023-01-29 at 10 39 46 PM" src="https://user-images.githubusercontent.com/31802656/215330052-2c151b5b-bb83-4d6e-a3b3-2e4c9fb548d1.png">

Reply:
I wonder why `Registered Nurse` isn't showing up in your DB. Everything from `degrees.csv` should be loaded in the db...

Reply:
Possibly a race condition in forEach...

Reply:
I refactored the seed data function. I think `forEach` with the implied sql ids meant that there was a race condition.

Try it again on a fresh DB, I think it's working. Will have to refactor the seed data for the other methods later